script/install-libseccomp.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -eux -o pipefail
+
+VERSION=2.5.1
+SHA256SUM="ee307e383c77aa7995abc5ada544d51c9723ae399768a97667d4cdb3c3a30d55"
+
+if [ -z ${@+x} ]; then
+	HOSTS=(
+		"x86_64-linux-gnu"
+		"arm-linux-gnueabi"
+		"arm-linux-gnueabihf"
+		"aarch64-linux-gnu"
+		"powerpc64le-linux-gnu"
+	)
+else
+	HOSTS="$@"
```

Comment:
Can probably remove the else now, no one is calling with a parameter this script

Reply:
Removed it